lib/kupong_integration/service.rb

```diff
@@ -95,11 +95,11 @@ def sanitize(phone)
     end
 
     def authorization
-      settings[:authorization]
+      settings[:authorization] || settings['authorization']
```

Comment:
Would it bet better convert the settings hash using `hash_with_indifferent_access` on initialize so we don't have to do this here and for `coupon_id`?